files/build/versions/build/build-sonic-slave-bullseye/versions-deb-bullseye-amd64

```diff
@@ -0,0 +1,25 @@
+libhiredis-dev==0.14.0-3~bpo9+1
```

Comment:
Are the files in files/build/versions checked in by mistake? We will not enable reproducible build on master branch.

Reply:
Removed

Reply:
I mean to remove all the files in files/build/versions.